Added new version of Korean translations for document, and some Miscellaneous changes!

pinkrabbit412 opened the pull request:
# New Korean translation is now here!
I’ve translated tweepy’s documents to Korean, to more people can use tweepy easily.
Previous translation was outdated, but I can’t modify or update it because I had to serve in the military in Korea(not North, but near(DMZ/GOP)). But now I’m discharged due to expiration of mandatory service period, so I’ve made new Korean translation document for tweepy.

## 1. Commit summary
A. Added: new version of Korean translation for reference document.
B. Added: krHTML_autoBuild.bat for debugging .po files at local devices (QoL).
C. Removed: unused document .po files and TRANSLATORS_ko_KR.txt.
 
## 2. What’s changed
### A. Added new version of Korean translation
 Translated next documents:
 > api.po, asyncstream.po, auth_tutorial.po, authentication.po, changelog.po, client.po, development.po, examples.po, exceptions.po, extended_tweets.po, faq.po, getting_started.po, index.po, install.po, models.po, pagination.po, parameters.po, stream.po, streaming.po

 Much of the translation has been modified to suit with the latest version. The terms and explanations used in documents were also unified as much as possible. There might be an loose translation for convinence at understanding, but It doesn’t interfere the original sentence’s intend.
### B. Added krHTML_autoBuild.bat for debugging .po files at local devices (QoL).
 Previously, we need to build Korean version of HTML with 2 commands to debug documents, but It’s kind a tiresome work. So I’ve added .bat file. This will be changed to improved batch files at future.
### C. Removed unused document .po files and TRANSLATORS_ko_KR.txt.
 I’ve removed unused document files (ex. streaming_how_to.po), and TRANSLATORS_ko_KR.txt(This file is also outdated, and overlaps with tweepy/tweepy → Insights → Contributors).



## 3. Known Issue
### A. Completed translation of examples.po, but it doesn't appear at final results.
Because it's not an important part (it's the title of the tab), this part will be revised later.
### B.  Some Duplicate label errors. Error logs below.
    C:\Users\pinkr\GitHub Repository\tweepy\docs\..\tweepy\asynchronous\streaming.py:docstring of tweepy.asynchronous.streaming.AsyncStream.filter:: WARNING: more than one target found for cross-reference 'List': tweepy.models.List, tweepy.List
    C:\Users\pinkr\GitHub Repository\tweepy\docs\..\tweepy\asynchronous\streaming.py:docstring of tweepy.asynchronous.streaming.AsyncStream.filter:: WARNING: more than one target found for cross-reference 'List': tweepy.models.List, tweepy.List
    C:\Users\pinkr\GitHub Repository\tweepy\docs\..\tweepy\asynchronous\streaming.py:docstring of tweepy.asynchronous.streaming.AsyncStream.filter:: WARNING: more than one target found for cross-reference 'List': tweepy.models.List, tweepy.List
    C:\Users\pinkr\GitHub Repository\tweepy\docs\..\tweepy\asynchronous\streaming.py:docstring of tweepy.asynchronous.streaming.AsyncStream.filter:: WARNING: more than one target found for cross-reference 'List': tweepy.models.List, tweepy.List
    C:\Users\pinkr\GitHub Repository\tweepy\docs\..\tweepy\asynchronous\streaming.py:docstring of tweepy.asynchronous.streaming.AsyncStream.sample:: WARNING: more than one target found for cross-reference 'List': tweepy.models.List, tweepy.List
    C:\Users\pinkr\GitHub Repository\tweepy\docs\..\tweepy\errors.py:docstring of tweepy.errors.HTTPException:15: WARNING: more than one target found for cross-reference 'List': tweepy.models.List, tweepy.List
    C:\Users\pinkr\GitHub Repository\tweepy\docs\..\tweepy\errors.py:docstring of tweepy.errors.HTTPException:21: WARNING: more than one target found for cross-reference 'List': tweepy.models.List, tweepy.List
    C:\Users\pinkr\GitHub Repository\tweepy\docs\..\tweepy\errors.py:docstring of tweepy.errors.HTTPException:27: WARNING: more than one target found for cross-reference 'List': tweepy.models.List, tweepy.List

## 4. Miscellaneous
### A. Recommended label: ‘Documentation’
### B. Translated by
 - 악동분홍토끼(@pinkrabbit412)
https://github.com/pinkrabbit412

Harmon758 commented on docs/locale/ko/LC_MESSAGES/api.po:
```diff
@@ -1,633 +1,2881 @@
-# Copyright (C) 2009-2020, Joshua Roesslein
-# This file is distributed under the same license as the Tweepy package.
-# 악동분홍토끼 <pinkrabbit412@daum.net>, 2019.
+# SOME DESCRIPTIVE TITLE.
```
This should probably be updated.

Harmon758 commented on docs/locale/ko/LC_MESSAGES/asyncstream.po:
```diff
@@ -0,0 +1,401 @@
+# SOME DESCRIPTIVE TITLE.
```
This should probably be updated.

Harmon758 commented on docs/locale/ko/LC_MESSAGES/asyncstream.po:
```diff
@@ -0,0 +1,401 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2009-2021, Joshua Roesslein
+# This file is distributed under the same license as the tweepy package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
```
This should probably be updated.

Harmon758 reviewed (changes requested):
Thanks for the PR!
Don't worry about the warnings about `List`. I'm aware of them, and they're unrelated to translation.

I think the batch file is fine for personal use, but I don't think it needs to be included in the library.

The first (`# SOME DESCRIPTIVE TITLE.`) and fourth (`# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.`) comment lines of most of the `.po` files should probably be updated.

There's also additions and changes to the Polish `.po` translation files. Could you remove those from the PR?

pinkrabbit412 commented:
Thanks for PR reviewing!
I've made some changes, so I re-requesting PR review.

### Additional changes
- Removed unnecessary batch file (krHTML_autoBuild.bat)
- 1st and 4th line of .po files modified appropriately
- Reverted Polish translation file to previous version
